### PR TITLE
Document openSUSE KMPS boo#1182641

### PR DESCRIPTION
**Pull request by lkocman**

### CODE REVIEW

**Comment by lkocman** [comment]
This is expected to be the last RN note for this release. Sorry for being late.

**Comment by KucharczykL** [comment]
Thank you for submitting the PR!

I think the explanation right now is a bit too detailed. In the release notes it is not necessary to explain all the details but it's important to let the user know what should they do.

**Comment by lkocman** [comment]
I left only the important part and skipped details that might not be that important or are otherwise duplicated across RN.

File: xml/release-notes.xml [hunk 1]
```diff
@@ -559,6 +559,23 @@
  <sect1 xml:id="driver-hardware">
   <title>Drivers and hardware</title>
   <para/>
+  <sect2 xml:id="driver-hardware-secure-boot-opensuse-kmps">
+   <!-- bsc#1182641 -->
+   <title>Secure Boot: &sle; kernel and openSUSE signed Kernel Module Packages</title>
+   <para>
+    Newly introduced <package>openSUSE-signkey-cert</package> is required for openSUSE KMPs like virtual-box,
```
Comment (KucharczykL) [inline xml/release-notes.xml:566]:
```suggestion
    The newly introduced <package>openSUSE-signkey-cert</package> package is required for openSUSE KMPs like <package>virtualbox</package>,
```

File: xml/release-notes.xml [hunk 2]
```diff
@@ -559,6 +559,23 @@
...
+    but only in secure boot mode. Package includes the certificate of
```
Comment (KucharczykL) [inline xml/release-notes.xml:567]:
```suggestion
    but only in Secure Boot mode. The package includes the certificate of
```

File: xml/release-notes.xml [hunk 3]
```diff
@@ -559,6 +559,23 @@
...
+    openSUSE signing key for signing kernel module file (.ko) in openSUSE KMP
```
Comment (KucharczykL) [inline xml/release-notes.xml:568]:
```suggestion
    openSUSE signing key for signing kernel module file (<literal>.ko</literal>) in openSUSE KMP
```

File: xml/release-notes.xml [hunk 4]
```diff
@@ -559,6 +559,23 @@
...
+    and calls mokutil help user to enroll certificate to MOK.
```
Comment (KucharczykL) [inline xml/release-notes.xml:569]:
```suggestion
    and calls <literal>mokutil</literal> to help user enroll the certificate to MOK.
```

File: xml/release-notes.xml [hunk 5]
```diff
@@ -559,6 +559,23 @@
...
+    So the openSUSE KMP can be verified by kernel.
```
Comment (KucharczykL) [inline xml/release-notes.xml:570]:
```suggestion
    This way, the openSUSE KMP can be verified by the kernel.
```

File: xml/release-notes.xml [hunk 6]
```diff
@@ -559,6 +559,23 @@
...
+   </para>
+   <para>
+    Systems without the base pattern which are subject to these KMPs are advised to get
+    <package>openSUSE-signkey-cert</package> installed manually. A system reboot is required.
```
Comment (KucharczykL) [inline xml/release-notes.xml:574]:
```suggestion
    If you do not have the base pattern installed and are using any of these KMPs, we recommend
    installing the <package>openSUSE-signkey-cert</package> package manually. A system reboot is required.
```

File: xml/release-notes.xml [hunk 7]
```diff
@@ -559,6 +559,23 @@
...
+    More information about this process and manual enrollment can be found on the following wiki page
```
Comment (KucharczykL) [inline xml/release-notes.xml:575]:
```suggestion
    More information about this process and manual enrollment can be found at
```

**Review by KucharczykL — APPROVED** [review]
I made some changes after your simplification. Looks good to me now.

**Comment by lkocman** [comment]
Thank you @KucharczykL merging.

**Pull request merged**